sonoff/sonoff_template.h

```diff
@@ -1217,6 +1225,22 @@ const mytmplt kModules[MAXMODULE] PROGMEM = {
      GPIO_USER,
      GPIO_USER,
      0
+  },
+    { "ARMTR Dimmr",     // ARMTRONIX Dimmer (ESP8266 w/ separate MCU dimmer)
+                       // https://www.amazon.com/gp/product/B07CTNSZZ8/ref=oh_aui_detailpage_o00_s00?ie=UTF8&psc=1
+     GPIO_USER,        // Virtual Button (controlled by MCU)
+     GPIO_USER,        // GPIO01 MCU serial control
+     GPIO_USER,
+     GPIO_USER,        // GPIO03 MCU serial control
+     GPIO_USER,
+     GPIO_USER,
+     0, 0, 0, 0, 0, 0, // Flash connection
+     GPIO_USER,
+     GPIO_USER,
+     GPIO_USER,        // GPIO14 Green Led
+     GPIO_USER,
+     GPIO_USER,
```

Comment:
all GPIO_USER? no key, no buttons, no leds, no relays, no pwm? So why not use Generic instead?

Reply:
Well, it is a new kind of module in the most physical sense. There are some buttons, I will try to find out where they are connected (although they are not accesible outside of the module and while powered by 230VAC you probably don't want to touch them anyway...)

When I now think about it, my recommendation would be to keep the extra module, as the hardware is physically different. But to not clutter everything up too much, remove the new RX and TX defines. Will propose another solution in an updated pull request.